start.html

```diff
@@ -50,31 +50,51 @@ <h3>Install monai</h3>
+                <p>Training and evaluation code examples for 3D medical image segmentation<br/> <a href="https://github.com/Project-MONAI/tutorials/tree/master/3d_segmentation/torch">Volumetric image segmentation examples <i class="far fa-caret-square-right"></i></a> </p>
+                <p>Training and evaluation code examples for 3D medical image segmentation (with PyTorch Ignite)<br/> <a href="https://github.com/Project-MONAI/tutorials/tree/master/3d_segmentation/ignite">Volumetric image segmentation examples with Ignite <i class="far fa-caret-square-right"></i></a> </p>
+                <p>Build a segmentation workflow (with PyTorch Ignite)<br/> <a href="https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/unet_segmentation_3d_ignite.ipynb">Segmentation workflow demo with Ignite <i class="far fa-caret-square-right"></i></a> </p>
+                <p>Build a segmentation workflow (with Catalyst framework)<br/> <a href="https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/unet_segmentation_3d_catalyst.ipynb">Segmentation workflow demo with Catalyst <i class="far fa-caret-square-right"></i></a> </p>
+            </div>
+            <div class="pure-u-1 pure-u-md-1-2">
+                <h3>Performance Acceleration</h3>
+                <p>Training and evaluation code examples with Distributed Training supports<br/> <a href="https://github.com/Project-MONAI/tutorials/tree/master/acceleration/distributed_training">Distributed training examples <i class="far fa-caret-square-right"></i></a></p>
+                <p>3D segmentation with Automatic Mixed Precision supports<br/> <a href="https://github.com/Project-MONAI/tutorials/blob/master/acceleration/automatic_mixed_precision.ipynb">Spleen segmentation tutorial with AMP supports <i class="far fa-caret-square-right"></i></a></p>
+                <p>Accelerate model training by caching methods<br/> <a href="https://github.com/Project-MONAI/tutorials/blob/master/acceleration/dataset_type_performance.ipynb">Comparisons between different loading methods <i class="far fa-caret-square-right"></i></a> </p>
+                <p>Fast training under MONAI features<br/> <a href="https://github.com/Project-MONAI/tutorials/blob/master/acceleration/fast_training_tutorial.ipynb">Fast training demo <i class="far fa-caret-square-right"></i></a> </p>
+                <p>Data-parallel training with multiple-GPUs<br/> <a href="https://github.com/Project-MONAI/tutorials/blob/master/acceleration/multi_gpu_test.ipynb">Multi-GPU training demo <i class="far fa-caret-square-right"></i></a> </p>
+                <p>GPU-based preprocessing in native PyTorch<br/> <a href="https://github.com/Project-MONAI/tutorials/blob/master/acceleration/transform_speed.ipynb">GPU-based pipeline demo <i class="far fa-caret-square-right"></i></a> </p>
+            </div>
+            <div class="pure-u-1 pure-u-md-1-2">
+                <h3>MONAI Modules</h3>
```

Comment:
Could you please help change to `Modules Demo`? I think no need to use `MONAI` here.

Reply:
I see, I've updated the code.